policy/README.md

```diff
@@ -0,0 +1,353 @@
+
+<a name="endpoints"></a>
+## Endpoints
+
+The provider-facing Policy API consists of the following two endpoints.
+
+`GET /policies`
+
+Parameters:
+
+| Name         | Type      | R/O | Description |
+| ----         | ----      | --- | ----------- |
+| `start_time` | timestamp | O   | Earliest effective date, default=effective now |
+| `end_time`   | timestamp | O   | Latest effective date, default=all future |
+
+Note: provider_id is an implicit parameter and will be encoded in the authentication key
+
+Returns: List of policy objects effective during the timespan described by greater than or equal to `start_time` and less than or equal to `end_time`
+
+Policies will be returned in order of effective date (see Policy Schema, below), with pagination as in Agency or Provider.
+
+`GET /policies/{id}`
+
+Returns: One policy object whose id is `id`
+
+Parameters: none
+
+`GET /geographies/{id}`
+
+Returns: One immutable GeoJSON FeatureCollection with an embedded UUID identifier property of `id`
+
+Parameters: none
+
+Note: Intentionally omitted `GET /geographies` until a compelling use-case can be identified.
+ 
+<a name="authoring"></a>
+## Authoring
+
+Creating, editing, and publishing policies may be performed via a variety of mechanisms, and are therefore not specified here.  Authoring tools would optionally provide schema extensions for tooling, including author, Provider-specificity, etc.  We may add a specific instance of such extensions in a later revision of this document, but at present that’s TBD.
+
+One mechanism may be via a source-control repository, where pull-requests to Policy objects are proposed, left open to public commentary, etc., and served as static content via the endpoints listed above.
+
+Another possibility would be a policy-editing REST API, where drafts of Policy objects are mutable, pending publication.  This would be the API for manual policy-creation with GUI tooling.  LADOT may propose a specific separate policy-editing API in the future.
+
+Certain policies could be fully dynamic, e.g. caps could be raised and lowered via algorithm on a day-to-day or even hour-to-hour basis.  No-fly-zones could be created in quasi-real-time in response to emergencies or road-closures. 
+
+Dynamic caps can also be implemented by replacing the “maximum” integer with a URL to a source for dynamic data.  This could be for provider-specific caps that go up and down, or for global caps e.g. “total 500 scooters at the coliseum”.  Dynamic data sources would be required to have historical data so that validating prior information, and downloaded dynamic data would have a time-to-live.
+
+<a name="compliance"></a>
+## Compliance
+
+A Compliance API will be described in a separate MDS specification.  In brief, it will take as inputs the MDS Agency data stream, the MDS geography data, and these MDS Policy objects and emit Compliance objects.  This work is in draft form but is closely informed by this Policy specification.
```

Comment:
same comment as above. This seems overly complex

Reply:
I'd like to leave this forward-pointer.  We will have the Compliance API spec soonish, once Policy firms up.